roles/smartgateway/templates/deployment.yaml.j2

```diff
@@ -47,15 +47,16 @@ spec:
         volumeMounts:
         - mountPath: /tmp
           name: socket-dir
-{% endif %}
-{% if service_type == 'events' %}
+{% else %}
       - name: smart-gateway
         image: '{{ events_container_image_path }}'
```

Comment:
The name `events_container_image_path` is misleading now. It comes from here: https://github.com/infrawatch/smart-gateway-operator/blob/sg3-bz1805468/roles/smartgateway/tasks/main.yml#L9

I think we should choose a different name, but for the moment I wanted to change as little as possible to help keep the merge simpler.